### PR TITLE
Fix: i18n po/mo not loaded

### LINKED ISSUE #660: i18n po/mo not loaded
**Describe the bug**
I translated Distributor in french but only strings in the plugin page but the translations are not loaded in the plugin setings it self:
distrubor-fr
distrubor-fr-settings

[distributor-fr_FR.zip](https://github.com/10up/distributor/files/5432900/distributor-fr_FR.zip)

**Environment information**
 - WordPress version: 5.5.1

### PR DESCRIPTION
**Pull request by dinhtungdu**
### Description of the Change
This PR loads textdomain properly, allowing distributor to ship translations with future release. Props @sebastienserre for reporting this.

### Verification Process
See #660 

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Applicable Issues
Fixes #660

### CODE REVIEW

**Comment by jeffpaul** [comment]
@dinhtungdu will want to make sure we confirm the checklist items on #267 for this PR.

File: distributor.php [hunk 1]
```diff
@@ -183,6 +183,8 @@ function() {
 		) {
 			\Distributor\Connections::factory()->register( '\Distributor\InternalConnections\NetworkSiteConnection' );
 		}
+
+		load_plugin_textdomain( 'distributor', false, dirname( plugin_basename( __FILE__ ) ) . '/lang' );
```
Comment (dkotter) [inline distributor.php:187]: So this loads from the `lang` directory but that isn't a directory that we currently have in Distributor. I get that this is setting us up for the future but may be worth adding that directory as part of this PR?

**Comment by dkotter** [comment]
@jeffpaul @dinhtungdu Left a comment on the PR but leaving it here as well. This PR loads a textdomain but doesn't actually add the `lang` directory (or add any translations). I understand this is just a single step in the right direction and actual translations can come later but I think we might want to add the `lang` directory as part of this PR, so it's there and ready to drop translations into.

**Comment by jeffpaul** [comment]
I believe the `lang` directory and POT file get built into the `stable` branch as part of the release action, but maybe that's worth having in `develop` and `trunk` as well?  Will defer to @helen, though may just be worth punting this if there's any discussion needed so as to not hold up the 1.6.2 release.

**Comment by dkotter** [comment]
> I believe the `lang` directory and POT file get built into the `stable` branch as part of the release action, but maybe that's worth having in `develop` and `trunk` as well? Will defer to @helen, though may just be worth punting this if there's any discussion needed so as to not hold up the 1.6.2 release.

Ah, that makes sense. But yeah, does bring up the point of how do people submit a PR with language files if there isn't a `lang` directory to put those files in (which was brought up on #660)

I'm fine with this PR but there's definitely more work around translations that needs done, so could argue for waiting on this until we have the other parts in place.

**Comment by jeffpaul** [comment]
I'm going to punt to 1.7.0 to give us time to chat about if/how we want to handle MO/PO files and other topics related to translations.

File: distributor.php (hunk 1 above)
Comment (dinhtungdu) [reply distributor.php:187]: @dkotter We have that directory in the `stable` branch. We build the pot file and put it into `lang` folder for the release.

**Comment by peterwilsoncc** [comment]
This was resolved in e6f1dd37194b7276d53b37e5a21d3a68fdb01fb2 for another PR, so closing this off.

**Pull request closed without merging**